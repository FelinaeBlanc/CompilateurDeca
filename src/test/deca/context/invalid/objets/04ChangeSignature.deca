// This test should not work as we cannot redefine a method whith an other signature

class A {
    int x;

    int getX() {
        return x;
    }
}

class B extends A {

    float getX() {
        return 12.3;
    }
}