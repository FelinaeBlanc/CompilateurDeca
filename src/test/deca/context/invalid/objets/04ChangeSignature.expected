Method getX exist in parent but don't have same return type
